### PR TITLE
Add [-quiet bool] option to exechealthz.go

### PR DESCRIPTION
**Pull request by ghost**
exechealthz.go generates a log message every -period.  For instance:

```
2015/08/01 00:56:44 Worker running nslookup kubernetes.default.svc.example.local localhost &> /dev/null
```

This change adds a -quiet option to disable the periodic messages.  It has a default value "false" so the cli is backwards compatible if omitting the flag.

### CODE REVIEW

**Comment by entropyfarms** [comment]
ok to test

--
Brandon

File: contrib/exec-healthz/exechealthz.go [hunk 1]
```diff
@@ -86,7 +87,9 @@ func (h *execWorker) start() {
 		select {
 		// If the command takes > period, the command runs continuously.
 		case <-ticker.C:
-			log.Printf("Worker running %v", *cmd)
+			if !quiet {
```
Comment (bprashanth) [inline contrib/exec-healthz/exechealthz.go]: Can you please make this a positive check? eg: 

verbose = flag(verbose, false)
if verbose { log.Printf() }

**Comment by bprashanth** [comment]
LGTM sans tiny nit, @entropyfarms thanks!

**Comment by entropyfarms** [comment]
Thanks Prasanth.

Never actually a Go coder but I felt the pull request would be more helpful (even if needed a little work) than just creating a "todo" issue.  

I'll get those changes out for review this weekend or when I am in the office early next week.  My kibana server is looking forward to the quietude. 

--
Brandon

**Comment by entropyfarms** [comment]
I signed it!

**Comment by mikedanese** [comment]
Can you please squash commits into one?

**Comment by bprashanth** [comment]
Kicked shippable since I couldn't get the logs from the last run, lets see

**Comment by bprashanth** [comment]
@entropyfarms really sorry for the delay. This just fell off my radar. Can you please rebase when you have time and @ me when you're ready?

**Comment by bprashanth** [comment]
The newest exechealthz image should have a -quiet flag (https://github.com/kubernetes/contrib/pull/498#issuecomment-187482306)

**Pull request closed without merging**